main/src/ca/uwaterloo/flix/language/phase/Typer.scala

```diff
@@ -1563,14 +1563,14 @@ object Typer {
         for {
           (constrs, tpe, _, eff) <- visitExp(exp)
           _ <- expectTypeM(expected = Type.Int32, actual = tpe, exp.loc)
-          resultTyp <- liftM(Type.mkChannel(elmType, loc))
+          resultTyp <- liftM(Type.mkSender(elmType, loc))
```

Comment:
This should return a pair `(Sender[t], Receiver[t]`). 

I think `Type.mkTuple` and then it should work.

Reply:
Yes, I originally implemented that, but that then requires all of the `par` code to be rewritten. So I thought it easier to do it this way.

But, thinking about it, I could instead rewrite the `par` code so that it uses `Concurrent/Channel` directly, which would that problem.